lib/mongoid/contextual/mongo.rb

```diff
@@ -454,6 +454,73 @@ def pluck(*fields)
         end
       end
 
+      # Get a hash of counts for the values of a single field. For example,
```

Comment:
I would add `"using the MongoDB aggregation pipeline"` here to make it explicit that it's a pipeline calculation. Although its an implementation detail, I think it's important for users to know how is works under-the-hood to understand the performance implications of using it on large collections.

Reply:
What would be the other option?

Reply:
For example, my (#pluck_each PR)[https://github.com/mongodb/mongoid/pull/5332], you could use that to do a relatively efficient tally in Ruby memory (using GETMORE commands, etc.)